.pre-commit-config.yaml

```diff
@@ -122,7 +117,6 @@ ci:
     - check-yaml
     - check-json
     - check-datafiles
-    - check-forbidden-words
```

Comment:
why we don't need this anymore

Reply:
Check was removed as `openai_api_base` is one of the forbidden words